akka-actor-typed/src/main/scala/akka/actor/typed/scaladsl/Behaviors.scala

```diff
@@ -29,6 +29,15 @@ object Behaviors {
   def setup[T](factory: ActorContext[T] => Behavior[T]): Behavior[T] =
     BehaviorImpl.DeferredBehavior(factory)
 
+  /**
+   * Support for stashing messages to unstash at a later timej.
+   */
+  def withStash[T](capacity: Int, factory: StashBuffer[T] => Behavior[T]): Behavior[T] =
```

Comment:
Should it perhaps be two param lists instead, some of the other behaviors that take factories do it like that allowing for:

```
Behaviors.withStash(100) { buffer => 
  Behaviors.etc
}
```

Reply:
Yep that reads nicer